src/compiler/compile/nodes/Element.ts

```diff
@@ -428,7 +428,7 @@ export default class Element extends Node {
 			if (attribute) {
 				const value = attribute.get_static_value();
 
-				if (value === '' || value === '#') {
+				if (value === '' || value === '#' || /^\W*?javascript/.test(value)) {
```

Comment:
The `?` here after the `*` doesn't do anything - and we should be testing for `javascript:`, not just `javascript`. There's nothing wrong with linking to paths that happen to begin with `javascript`.